temporal-sdk/src/main/java/io/temporal/client/UpdateOptions.java

```diff
@@ -154,6 +154,13 @@ public void validate() {
     }
   }
 
+  void validateWaitForCompleted() {
+    if (waitForStage != null && waitForStage != WorkflowUpdateStage.COMPLETED) {
+      throw new IllegalArgumentException(
+              "waitForStage must be unspecified or " + WorkflowUpdateStage.COMPLETED);
+    }
+  }
```

Comment:
So I added this because I think users would rather ensure that the UwS call waits for "completed" when they call `executeUpdateWithStart`, than have it be "accepted" (accidentally) and potentially impacting their experience (since an additional polling step might be needed).

This also seems better than silently setting it, but I don't feel strongly about this.

Reply:
Whatever we decide, the behavior for the "execute update" API(s) should do the same.